src/tasks/minions/miningActivity.ts

```diff
@@ -119,7 +132,7 @@ export default class extends Task {
 			user,
 			channelID,
 			str,
-			['mine', [quantity, ore.name], true],
+			['mine', [duration > 0.9 * user.maxTripLength('Mining') ? null : quantity, ore.name], true],
```

Comment:
This isn't enough to prevent quantity from 'overflowing' with some bad RNG. (after quantity is implemented obviously)
The +mine QTY ore command needs to handle excess gracefully, because we don't want ppl 'cheating' the system retrying. Maybe it hides the amount as well as gives a fake time. Just say, 'mining for x minutes'

Reply:
this only makes it so that it always tries to do max trip if duration is above 90%, this because otherwise some new trips will be stuck with lower quantities even tho the person pressing the ''Y'' want a full 30-40 min trip. But with the other masking that suggested earlier I think this is okay.